src/openforms/forms/tests/variables/test_initial_values.py

```diff
@@ -0,0 +1,34 @@
+from django.test import TestCase
+from django.utils import timezone
+
+from freezegun import freeze_time
+
+from openforms.forms.constants import FormVariablesInitialValues, FormVariablesSources
+from openforms.forms.tests.factories import FormVariableFactory
+
+
+class FormVariablesTests(TestCase):
+    def test_get_initial_value(self):
+        static_variable = FormVariableFactory.create(
+            source=FormVariablesSources.static,
+            initial_value=FormVariablesInitialValues.today,
+        )
+        with self.subTest(part="static variable today"):
+            with freeze_time("2021-07-16T21:15:00Z"):
+                self.assertEqual(
+                    timezone.now().isoformat(), static_variable.get_initial_value()
```

Comment:
use the actual literal expected value in tests, instead of duplicating the implementation. `timezone.now().isoformat()` is too much thinking to figure out what the expected result is. `"2021-07-16T21:15:00Z"` is immediately obvious.